app/src/main/java/com/sdpteam/connectout/event/nearbyEvents/filter/EventsFilterDialog.java

```diff
@@ -71,7 +73,7 @@ private SeekBar.OnSeekBarChangeListener seekBarHandler(View view) {
             public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
                 final TextView progressLabel = view.findViewById(R.id.events_filter_seekbar_value);
                 if (progress == 100) { // acts like a disabled filter
```

Comment:
I would advise against the use of magic number, sure here 100% represent completion, 
maybe using the `SEEKBAR_VALUE` here could help.